### DISCUSSION TITLE
Vector swizzles

### DISCUSSION THREAD

**Discussion by DoeringChristian** [mitsuba-renderer/mitsuba3#355] [Feature requests]
Hi,
Is it possible to implement swizzling operations for vector and point types in the python api using properties? For example:
```python
v = mi.Vector3f(1, 2, 3)
u = v.xy
```
 I think this might be usefull since I sometimes have the case where they would be helpfull.
Thanks for regarding my request.

**Comment by Speierers** [comment]
This exists in C++ as `Vector2f uv = dr::head<2>(v);`. We could think of adding Python bindings for this function, I will add this to our TODO list.
